@AH-64D Apache Longbow/Addons/fza_ah64_common/functions/fn_debugDrawLine.sqf

```diff
@@ -38,4 +38,4 @@ switch (_col) do {
     };
 };
 
-drawLine3D [_heli modelToWorldVisual _p1, _heli modelToWorldVisual _p2, _col];
\ No newline at end of file
+debugDrawLine3D [_heli modelToWorldVisual _p1, _heli modelToWorldVisual _p2, _col];
```

Comment:
slight error with the replace here

Reply:
Agreed, I guess this was just a too general find-replace?

Reply:
Fixed!